src/process/message_processor.go

```diff
@@ -1,48 +1,94 @@
 package process
 
 import (
+	"BGPAlert/analyze"
 	"BGPAlert/common"
 	"errors"
 	"fmt"
 	"time"
 )
 
-// Constantly reads channel of messages and stores them in Window objects to send through windowChannel for analysis
-func ProcessBGPMessages(msgChannel chan common.BGPMessage, windowChannel chan common.Window) error {
-	var bucketMap = make(map[time.Time][]common.BGPMessage)
+// Constantly read messages from channel, build up windows with frequency maps, calling analysis on windows when full
+func ProcessBGPMessages(msgChannel chan common.BGPMessage) error {
+	maximumBuckets := 43
```

Comment:
why 43?

Reply:
or more specifically, why not 42?

Reply:
also: could this/should this be configurable?